pymchelper/axis.py

```diff
@@ -108,3 +109,10 @@ class AxisId(IntEnum):
     z = 2
     diff1 = 3
     diff2 = 4
+
+
+class AxisDescription(NamedTuple):
```

Comment:
Why doing import of NamedTuple ?
Take a look here https://docs.python.org/3/library/collections.html#collections.namedtuple

I think the suggested method is to use `namedtuple` method

Reply:
Good point, we can stick with typing module approach, I like it.

By the way - maybe we should use simply a data class instead of named tuple ?

I think I've used data tuple many years ago as a substitute of dataclasses, which were not mature enough that time.